### PR TITLE
Chunk the MSETs to avoid large blocking redis for significant amounts…

### PR DESCRIPTION
**Pull request by ajordens**
… of time when indexing large accounts

- Will be selectively testing this and watching the SLOWLOG GET output for improvements (image caching is the biggest offender right now)
- Test cases use a maxMsetSize = 2

### CODE REVIEW

**Comment by ajordens** [comment]
PTAL @cfieber.

Looking to test this to see if it smooths out redis operations when large index operations are happening.

File: cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cache/RedisCache.java [hunk 1]
```diff
@@ -56,11 +57,13 @@
...
     private final JedisSource source;
     private final ObjectMapper objectMapper;
+    private final int maxMsetSize;
 
-    public RedisCache(String prefix, JedisSource source, ObjectMapper objectMapper) {
+    public RedisCache(String prefix, JedisSource source, ObjectMapper objectMapper, int maxMsetSize) {
         this.prefix = prefix;
         this.source = source;
         this.objectMapper = objectMapper.disable(SerializationFeature.WRITE_NULL_MAP_VALUES);
+        this.maxMsetSize = maxMsetSize;
```
Comment (cfieber) [inline cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cache/RedisCache.java]: should validate that this is an even number

Comment (cfieber) [inline cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cache/RedisCache.java]: I see below where it is correcting a potentially bad value, I'd just roll that up into this constructor, or throw an exception here if it is invalid

**Comment by cfieber** [comment]
LGTM

The build failures look maybe non-transient
````

com.netflix.spinnaker.cats.cache.CompositeCacheSpec > identifiers behaviour FAILED
    org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'com.netflix.spinnaker.cats.cache.CompositeCache@1710a332' with class 'com.netflix.spinnaker.cats.cache.CompositeCache' to class 'com.netflix.spinnaker.cats.cache.WriteableCache'
com.netflix.spinnaker.cats.provider.DefaultProviderCacheSpec > identifiers behaviour FAILED
    org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'com.netflix.spinnaker.cats.provider.DefaultProviderCache@38e0e8a7' with class 'com.netflix.spinnaker.cats.provider.DefaultProviderCache' to class 'com.netflix.spinnaker.cats.cache.WriteableCache'
````

**Pull request merged**